README.md

```diff
@@ -153,16 +190,19 @@ When submitting a pull request:
 
 Theme documentation and demo pages can be found in the [`/docs`](docs) if submitting improvements, typo corrections, etc.
 
+<a id="development"></a>
 ## Development
 
 To set up your environment to develop this theme, run `bundle install`.
 
-To test the theme, run `bundle exec rake preview` and open your browser at `http://localhost:4000/test/`. This starts a Jekyll server using content in the `test/` directory. As modifications are made to the theme and test site, it will regenerate and you should see the changes in the browser after a refresh.
+To test the theme, run `bundle exec rake preview` and open your browser at [`http://localhost:4001/test/`](http://localhost:4001/test/). This starts a Jekyll server using content in the `test/` directory. As modifications are made to the theme and test site, it will regenerate. Note that you will see the changes in your browser only after doing a manual page refresh.
```

Comment:
Still don't quite get the point of this change. The wording seems fine, though.

Reply:
See bullet 4 in my PR description, where I explained it:

> 4. Update Rakefile to serve jekyll on port 4001 instead of 4000, so as to avoid conflicts with [NoMachine](https://www.nomachine.com/). See [my answer here](https://stackoverflow.com/a/65512475/4561887) for more background and info on this.

For anyone who uses NoMachine, which is a very popular remote login tool, it uses port 4000 too and conflicts with the web server during website testing. This fixes that. For anyone else using NoMachine, it solves their problem without them having to search around a bunch to figure it out, and for everyone else it causes no harm, so is overall a useful change.

Reply:
I haven't even heard about NoMachine (I use SSH and VS Code). It's not convincing to change the default port because of a software that has no relation with Jekyll. FYI: I have `localhost:4000` saved in my browser bookmark for quick access, and this is going to break it.